client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -50,22 +50,34 @@ export const receiveCommentSuccess = ( store, action, next, response ) => {
 	} );
 };
 
-export const receiveCommentError = ( store, action ) => {
-	const site = getReaderSite( store.getState(), action.siteId );
+export const receiveCommentError = ( { dispatch, getState }, { siteId, commentId } ) => {
+	const site = getReaderSite( getState(), siteId );
 	const siteName = getReaderSiteName( { site } );
 
-	store.dispatch(
-		errorNotice(
-			translate( 'Failed to retrieve comment for site “%(siteName)s”', {
-				args: { siteName },
-			} ),
-		),
-	);
+	if ( siteName ) {
+		dispatch(
+			errorNotice(
+				translate( 'Failed to retrieve comment for site “%(siteName)s”', {
+					args: { siteName },
+				} )
+			)
+		);
+	} else {
+		const rawSite = getRawSite( getState(), siteId );
+		const error =
+			rawSite && rawSite.name
+				? translate( 'Failed to retrieve comment for site “%(siteName)s”', {
+					args: { siteName: rawSite.name },
+				} )
+				: translate( 'Failed to retrieve comment for your site' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Failed to retrieve comments for your site' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).